tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -598,15 +600,11 @@ def test_multi_field_orderby(self):
             for v_rating in ("good", "meh", "poor"):
                 # count decides the cardinality of this distribution bucket
                 for value in [123.4] * count:
-                    self.store_metric(
-                        org_id=self.organization.id,
-                        project_id=self.project.id,
+                    self.store_performance_metric(
                         type="distribution",
```

Comment:
We can probably figure out the `type` from the MRI, I would just add logic in `_store_metric_in_metrics_layer` to deduce the type from the MRI

Reply:
Yeah, we could also do that